fem/fespace.hpp

```diff
@@ -365,30 +417,62 @@ class FiniteElementSpace
        boundary. */
    void BuildNURBSFaceToDofTable() const;
 
-   /// Bit-mask representing a set of orders needed by an edge/face.
-   typedef std::uint64_t VarOrderBits;
-   static constexpr int MaxVarOrder = 8*sizeof(VarOrderBits) - 1;
+   void SetVarOrderLocalDofs();
```

Comment:
Document.

Reply:
Done in fc63c5e90b5ace705c64319123d3715d1dbe250f